tools/plugins/com.liferay.ide.project.ui/plugin.xml

```diff
@@ -1890,4 +1890,54 @@
 		>
 		</include>
 	</extension>
+	<extension
```

Comment:
hey @jtydhr88  and @Seiphon   can we move all of these files to a new plugin `com.liferay.ide.lsp.core` and `com.liferay.ide.lsp.ui`?

Reply:
OK， will do it